tracer/build/_build/Build.Utilities.cs

```diff
@@ -220,7 +230,6 @@ partial class Build
                             .Select(x => x.ToString())
                             .ToList();
 
-
             var integrations = GenerateIntegrationDefinitions.GetAllIntegrations(assemblies);
 
             GenerateIntegrationDefinitions.Run(integrations, TracerDirectory);
```

Comment:
In the lines below, should the dependabot stuff be removed?

Reply:
🤦‍♂️ Yes, they _were_ deleted at one point, I swear! Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2117/commits/71c7524c92bb6d67f3c9fd403a643901c74f9ce0